GameServerLib/Scripting/CSharp/CSharpScriptEngine.cs

```diff
@@ -23,22 +31,39 @@ public CSharpScriptEngine()
             _logger = LoggerProvider.GetLogger();
         }
 
-        public bool LoadSubdirectoryScripts(string folder)
+        public CompilationStatus LoadSubdirectoryScripts(string folder)
         {
             var basePath = Path.GetFullPath(folder);
             var allfiles = Directory.GetFiles(folder, "*.cs", SearchOption.AllDirectories).Where(pathString =>
             {
                 var fileBasePath = Path.GetFullPath(pathString);
                 var trimmedPath = fileBasePath.Remove(0, basePath.Length);
                 var directories = trimmedPath.ToLower().Split(Path.DirectorySeparatorChar);
-                if (directories.Contains("bin") || directories.Contains("obj")) return false;
+                if (directories.Contains("bin") || directories.Contains("obj"))
+                {
+                    return false;
+                }
+                if (pathString.Contains("AssemblyInfo.cs"))
```

Comment:
Why not adding it to the previous `if`?

Reply:
I wasn't happy with the length of the line, I'll combine them.